apps/base-docs/tutorials/docs/4_smart-wallet-and-eoa-with-onchainkit.md

```diff
@@ -0,0 +1,275 @@
+keywords:
+  [
+    JSON RPC,
+    RainbowKit,
+    OnchainKit,
+    frontend,
+    Next.js,
+    Base,
+    blockchain development,
+    dApps,
+    Base,
+    Base network,
+    blockchain connection,
+    smart contract development,
+    smart wallet,
+    Coinbase smart wallet,
+    eoa,
+  ]
+tags: ['account abstraction', 'frontend']
+difficulty: beginner
+hide_table_of_contents: false
+displayed_sidebar: null
+---
+
+The [Coinbase Smart Wallet] is a great way to onboard new users to onchain apps, and offers a number of experiential improvements to existing crypto users as well. As we're in a period of adoption and transition, a pain point has developed where a user with both the Smart Wallet and a browser-extension EOA can't always select the wallet they want while connecting to an app.
+
+In this tutorial, you'll learn how to more easily improve that experience by using [Wallet] component from [OnchainKit] to connect your users to your app.
+
+---
+
+## Objectives
+
+By the end of this tutorial, you should be able to:
+
+- Create a connection experience using the `Wallet` component from [OnchainKit]
+- Customize the `Wallet` component to allow your users to connect from a chosen list of wallets
+- Give users the option to select the [Coinbase Smart Wallet] or EOA while connecting to your app with RainbowKit
+
+---
+
+## Prerequisites
+
+### Be familiar with modern, frontend web development
+
+In this tutorial, you'll be working with a React frontend built with [Next.js]. While you don't need to be an expert, we'll assume that you're comfortable with the basics.
+
+### Possess a general understanding of the EVM and smart contract development
+
+This tutorial assumes that you're reasonably comfortable writing basic smart contracts. If you're just getting started, jump over to our [Base Learn] guides and start learning!
+
+### Understand providers and connectors
+
+You'll need to be familiar with how to connect an onchain app to the blockchain with a provider. If you're not, start with the [Onchain App Development] section of [Base Learn], or at least complete the tutorial [Introduction to Providers].
+
+### Coinbase Wallets
+
+You need to have both the [Coinbase Wallet] and [Coinbase Smart Wallet] for this tutorial. You need to set up the [Coinbase Wallet] in advance, but you can create a smart wallet during the tutorial.
+
+---
+
+## The Default Experience
+
+Begin by using [An Onchain App in 100 Components] as a template to quickstart a new app. Click the green `Use this template` button, create a new repo, download it, install dependencies, and run the app.
+
+```bash
+bun install
+bun run dev
+```
+
+The demo app contains sections showing a number of [OnchainKit] components, including the [Wallet]. Click `Connect Wallet` in your browser with the Coinbase wallet active, and you'll get the expected experience for an EOA wallet user with the browser extension.
+
+![Default eoa](../../assets/images/smart-wallet/onchainkit-default-eoa.png)
```

Comment:
Images don't match what I see on the latest onchainkit site running locally... 
<img width="958" alt="Screenshot 2024-08-09 at 1 02 18 PM" src="https://github.com/user-attachments/assets/9117aebf-2281-48ea-9452-1934f89c3c53">